nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -104,11 +105,23 @@
     HAVE_TE = False
 
 
-def get_specs(spec_name):
-    name_spec_dict = {"": get_gpt_layer_with_transformer_engine_spec(), "megatron_falcon_gpt": get_falcon_layer_spec()}
-    if spec_name not in name_spec_dict:
+@cache
+def mcore_supports_moe() -> bool:
+    mcore_version = packaging.version.Version(version("megatron-core"))
```

Comment:
I'm not 100% percent this is the correct package name, but this https://pypi.org/project/megatron-core/ seemed up-to-date.